SDKBuildScripts/Update-PlayFabSdk.ps1

```diff
@@ -0,0 +1,293 @@
+        {
+            # If either cloud or vertical is not provided, we want to remove the leading or trailing dot.
+            $cloudVertical = "$Vertical.$Cloud".TrimEnd(".")
+            $ApiSpecPfUrl = "https://$($cloudVertical).playfabapi.com/apispec"
+        }
+    }
+
+    if($PSCmdlet.ParameterSetName -eq "ApiSpecPath")
+    {
+        $apiSpecSource = "-apiSpecPath"
+
+        if($ApiSpecPath -and $ApiSpecPath -ne "default")
+        {
+            if(!(Test-Path $ApiSpecPath))
+            {
+                throw "Unable to find ApiSpecPath '$ApiSpecPath'."
+            }
+
+            $apiSpecSource += " $ApiSpecPath"
+        }
+    }
+    elseif($PSCmdlet.ParameterSetName -eq "ApiSpecPfUrl")
+    {
+        $apiSpecSource = "-apiSpecPfUrl"
+
+        if($ApiSpecPfUrl -and $ApiSpecPfUrl -ne "default")
+        {
+            if(![Uri]::IsWellFormedUriString($ApiSpecPfUrl, "Absolute"))
+            {
+                throw "You must specify a valid URL for ApiSpecPfUrl.  Unable to parse '$ApiSpecPfUrl' as a URL."
+            }
+
+            $apiSpecSource += " $ApiSpecPfUrl"
+        }
+    }
+    else
+    {
+        $apiSpecSource = "-apiSpecGitUrl"
+
+        if($ApiSpecGitUrl -and $ApiSpecGitUrl -ne "default")
+        {
+            $apiSpecSource += " $ApiSpecGitUrl"
+        }
+    }
+
+    Write-Verbose "API Spec Source: $apiSpecSource"
+}
+
+process
+{
+    foreach($sdkName in $SdkNames)
+    {
+        if(!$TargetSource)
+        {
+            $TargetSource = $sdkTargetSrcMap[$sdkName]
+        }
+
+        $destPath = Join-Path $sdksPath $sdkName
+
+        if(!(Test-Path $destPath))
+        {
+            $repoPath = "https://github.com/PlayFab/$sdkName"
+
+            if($PSCmdlet.ShouldProcess(
+                "Cloning SDK Repository for $sdkName into '$destPath'.",
+                "Would you like to clone the $sdkName repository from $($repoPath) into '$destPath'?",
+                "Unable to find $sdkName repository")
+              )
+            {
+                git clone $repoPath $destPath
+            }
+        }
+
+        # Add any overrides for a specific SDK
+        if($sdkName -eq "UE4")
+        {
+            # We only set this value if it wasn't explicitly provided.
+            if(!$NonNullable.IsPresent)
+            {
+                $NonNullable = $true
+            }
+        }
+
+        if(!$KeepSource -and (Test-Path $destPath))
+        {
+            Remove-Item $destPath -Recurse -Include *.as, *.cpp, *.cs, *.h, *.java, *.js, *.lua, *.m, *.php, *.py, *.ts
+        }
+
+        $buildIdentifier = ""
+        if($env:NODE_NAME)
+        {
+            $buildIdentifier = "-buildIdentifier JBuild_$($sdkName)_($env:NODE_NAME)_$($env:EXECUTOR_NUMBER)"
+        }
+
+        $sdkGenArgs = ""
+        if($NonNullable)
+        {
+            $sdkGenArgs += "-flags nonnullable "
+        }
+
+        if($Beta)
+        {
+            $sdkGenArgs += "-flags beta "
```

Comment:
We need a condition where if it's both, the sdkGenArgs are
-flags beta nonnullable
(It won't properly handle -flags beta -flags nonnulable, the new nonnullable will clobber the earlier beta declaration.